Unable to build from source

I am not able to build the current version.
Getting the following error:
pytorch/aten/src/TH/generic/THTensorLapack.c:860:7: warning: unused variable ‘n’ [-Wunused-variable]
   int n = ra__->size[1];
       ^
make[2]: *** [src/ATen/CMakeFiles/ATen.dir/__/TH/THTensor.cpp.o] Error 1
make[1]: *** [src/ATen/CMakeFiles/ATen.dir/all] Error 2
make: *** [all] Error 2

I am using Anaconda as directed in the readme.  I have created a conda environment and have set:
 export CMAKE_PREFIX_PATH="/u/anaconda2/envs/pyt4/"
As specified here: https://stackoverflow.com/questions/47799803/installing-from-source-within-anaconda-environment

Thanks

what compiler do you have? and what OS are you using?

The issue is this:

```
/u/pytorch/aten/src/THS/generic/THSTensor.cpp:357:98: error: expected ‘)’ before ‘PRId64’
   THArgCheck(d1 < nDimI && d2 < nDimI, 0, "Transposed dimensions should be sparse. Got nDimI: %" PRId64 ", d1: %" PRId64 ", d2: %" PRId64, nDimI, d1, d2);
```

--- a/aten/src/TH/THGeneral.h.in
+++ b/aten/src/TH/THGeneral.h.in
@@ -1,6 +1,8 @@
 #ifndef TH_GENERAL_INC
 #define TH_GENERAL_INC
 
+#define __STDC_FORMAT_MACROS
+
 #include <stdint.h>
 #include <stdlib.h>
 #include <stdio.h>

